nvflare/__init__.py

```diff
@@ -18,7 +18,5 @@
 
 # https://github.com/microsoft/pylance-release/issues/856
 
-from nvflare.app_common.executors.script_executor import ScriptExecutor
-from nvflare.app_common.workflows.fedavg import FedAvg
-from nvflare.job_config.fed_job import FedJob, FilterType
+from nvflare.job_config.api import FedJob, FilterType
```

Comment:
FilterType is not defined in job_config.api suggest change to
`from nvflare.job_config.defs import FilterType`